frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -377,11 +377,15 @@ const getRolloutStatus = (
   current: PodControllerOverviewItem,
   previous: PodControllerOverviewItem,
 ): boolean => {
-  const strategy = _.get(dc, ['spec', 'strategy', 'type']);
+  const {
+    spec: {
+      strategy: { type: strategy },
```

Comment:
I believe a test just made this an error (unless I misread the test failure).

Reply:
Yes, It did because Mock data doesn't have a strategy property defined in them. updated mock data in https://github.com/openshift/console/pull/2496/commits/3b8977c4a7be891a8440386ebc7ebd9609b646f1